pkg/providers/v1/instances_v2.go

```diff
@@ -83,17 +83,24 @@ func (c *Cloud) getAdditionalLabels(ctx context.Context, zoneName string, instan
 	// If topology labels are already set, skip.
 	if _, ok := existingLabels[LabelNetworkNodePrefix+"1"]; !ok {
```

Comment:
I think this is fine for now, but we may need to revisit this if the topology of an instance can change over time with a node stop + start.